### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by marcosfpr** [matchup-ir/match_up_api#1]
Translate README file
Create all documentation for this API

**Comment by GreatGhostsss** [comment]
If the API needs REST documentation, I'd love to help 🙂

**Comment by marcosfpr** [comment]
All help is very welcome. I was currently working on another project, but it will be a pleasure to help with this one again.

 There is a lot to be done and things that can be improved, documentation is one of them. It would be great if you want to do it :-)

If you have some question about the API, just type here and i'll help 👍

**Comment by Anamika1627** [comment]
Hello. I have just started learning about REST API documentation and wanted to know if I can assist @GreatGhostsss in the documentation?

This might help the work get finished soon and I will also be able to learn more about REST APIs and their documentation process.

**Comment by Anamika1627** [comment]
Hello. I have just started learning about REST API documentation and wanted to know if I can assist @GreatGhostsss in the documentation?

This might help the work get finished soon and I will also be able to learn more about REST APIs and their documentation process.

**Comment by marcosfpr** [comment]
Yeah, all help is veery welcome. Feel free to make changes to the project that you think can help :-)
